system/modules/core/library/Contao/Model.php

```diff
@@ -891,13 +937,22 @@ public static function findOneBy($strColumn, $varValue, array $arrOptions=array(
 	 */
 	public static function findBy($strColumn, $varValue, array $arrOptions=array())
 	{
+		$blnModel = false;
+
+		$arrColumn = (array) $strColumn;
```

Comment:
The naming of `$strColumn` is now misleading, so is the phpDoc.
This should be changed to reflect the new behaviour.